create_board.py

```diff
@@ -0,0 +1,53 @@
+from random import randrange, choice
+from string import ascii_lowercase
+
+
+"""
+creates board file
+"""
+
+def create_board(file_path, board_size=100, number_of_snakes=10, number_of_ladders=10, number_of_players=3):
```

Comment:
Why take default number_of_snakes/ladders/players if it is not part of the requirement?

It is expected that you do not take any assumptions unless the requirement itself is vague. Here it was specifically mentioned that you'll have to take them as input.